test/e2e/persistent_volumes.go

```diff
@@ -555,7 +560,7 @@ func makeWritePod(ns string, pvcName string) *api.Pod {
 					Name:    "write-pod",
 					Image:   "gcr.io/google_containers/busybox:1.24",
 					Command: []string{"/bin/sh"},
-					Args:    []string{"-c", "touch /mnt/SUCCESS && exit 0 || exit 1"},
+					Args:    []string{"-c", "touch /mnt/SUCCESS && (id -G | grep -P '\b777\b' && exit 0 || exit 1"},
```

Comment:
The "exit 0 || exit 1" is unnecessary -- grep already exits with a 0 or a 1 depending on if it finds a match.